blueprint/src/content.tex

```diff
@@ -203,9 +256,37 @@ \section{MainTheorem}
 \begin{lemma}\label{char_fun_FinInter}
   \lean{char_fun_FinInter}
   \uses{FinInter₀,eq_FinInter₀,toInt,char_fun}
+  \leanok
   The characteristic function of the intersection of a family of finite sets indexed by \verb|β| is equal to the product of their characteristic functions.
 \end{lemma}
 
+\begin{proof}
+  \leanok
+  Let $\mathcal{F} = \{A_1, A_2, \ldots, A_n\}$ be a collection of sets and let $B = \bigcap_{A \in \mathcal{F}} A$ be the intersection of these sets. Consider the characteristic function $\chi_B(x)$ for any element $x$. We wish to establish the following equivalence:
+
+  \begin{equation*}
+
```

Comment:
Do not leave blank here. It will cause failure of building.

Reply:
Done